app/javascript/react/components/MonstersIndex.js

```diff
@@ -1,8 +1,10 @@
-import React, { useState, useEffect} from "react"
+import React, { useState, useEffect, useCallback} from "react"
 import MonsterTile from "./MonsterTile"
 const MonstersIndex = () => {
 
   const [fetchedMonsters, setFetchedMonsters] = useState([])
+  const [state, updateState] = useState();
+  const forceUpdate = useCallback(() => updateState({}), []);
```

Comment:
An interesting approach, but forcing an update (render) is not in the spirit of React. Take a look at my previous comment on the fetch request for another approach to updating state